src/components/control-bar/ControlBar.js

```diff
@@ -142,7 +124,8 @@ export default class ControlBar extends Component {
   getChildren() {
     const children = React.Children.toArray(this.props.children);
     const defaultChildren = this.props.disableDefaultControls ? [] : this.getDefaultChildren();
-    return mergeAndSortChildren(defaultChildren, children, this.props);
+    const { className: _, ...parentProps } = this.props; // remove className
```

Comment:
Do you mean it's unused? But `className` is not used either. I think `_` is a convention name when you have to name a variable but not gonna use, so you don't pollute current scope.

Reply:
I'm OK with calling it just `className` here. But in the similar usage in `mergeAndSortChildren`, I prefer to rename it to `_order`, since it's a longer function. Sounds good?